src/tdastro/astro_utils/snia_utils.py

```diff
@@ -235,9 +235,9 @@ def compute(self, graph_state, rng_info=None, **kwargs):
         graph_state : `GraphState`
             An object mapping graph parameters to their values. This object is modified
             in place as it is sampled.
-        rng_info : `dict`, optional
-            A dictionary of random number generator information for each node, such as
-            the JAX keys or the numpy rngs.
+        rng_info : numpy.random._generator.Generator, optional
```

Comment:
Should it be? Or people'd understand that `optional` usually mean that it can be `None`?
```suggestion
        rng_info : numpy.random._generator.Generator or None, optional
```